### PR TITLE
📝 Fix grammar and clearify instructions

### PR DESCRIPTION
**Pull request by paldepind**
This PR fixes some grammar and clarifies a few things in the instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,7 +129,7 @@ struct FootballMatchApp: Widget {
 
   ```dart
   final _liveActivitiesPlugin = LiveActivities();
-  _liveActivitiesPlugin.init(appGroupId: "YOUR_CREATED_APP_ID");
+  _liveActivitiesPlugin.init(appGroupId: "YOUR_GROUP_ID");
```
Comment (paldepind) [inline README.md:132]: Previously this id was referred to both as "YOUR_CREATED_APP_I" and as "YOUR_GROUP_ID". For consistency and clarity, I've renamed all instances to the same name.

File: README.md [hunk 2]
```diff
@@ -66,7 +66,9 @@ You need to **implement** in your Flutter iOS project a **Widget Extension** & d
 
   <img alt="enable live activities xcode" src="https://raw.githubusercontent.com/istornz/live_activities/main/images/tutorial/enable_live_activities.gif" width="700px" />
 
-  - Create App Group for both ```Runner``` & your ```Widget Extension```.
+  - Add the "App Group" capability for both ```Runner``` and your widget extension.
+    After you add the capability, check the checkmark next to the text field that contains an identifier of the form `group.example.myapp`.
+    This identifier will be used later and refered to as `YOUR_GROUP_ID`.
```
Comment (paldepind) [inline README.md:71]: This is visible in the gif, but for people who are mostly reading the text (like me) I think it's helpful to spell it out as things will be broken later if this step is missed.

**Comment by istornz** [comment]
Hi @paldepind, thanks a lot for your PR 🥇!

**Comment by paldepind** [comment]
You're welcome. And thank you for creating this plugin 👍

**Pull request merged**